### PR TITLE
Events replay system

### PR DESCRIPTION
**Pull request by jpmonettas**
Not meant to be merged yet

### CODE REVIEW

File: src/ethlance/server/core.cljs [hunk 1]
```diff
@@ -88,6 +88,8 @@
                  :write-events-into-file? true
                  :file-path "ethlance-events.log"}
 
+   :event-store {:store-file "event-store.log"}
```
Comment (madvas) [inline src/ethlance/server/core.cljs:91]: Here you refer to it as :event-store, but namespace is `ethlance.server.events-store` (eventS). Please make ti consistent as event-store everywhere

Comment (jpmonettas) [reply src/ethlance/server/core.cljs]: Done!

Comment (madvas) [reply src/ethlance/server/core.cljs]: hmm but I wrote make it as event-store, not events-store

Comment (jpmonettas) [reply src/ethlance/server/core.cljs:91]: Meh sorry, read it the other way around, changed everything to event-store

**Pull request merged**